app/views/admin/categories/index.html.erb

```diff
@@ -1,4 +1,6 @@
 <% @categories.each do |category| %>
   <p><h4> <%= category.id %> <%= category.name %></h4></p>
   <%= link_to "Edit Category", edit_category_path(category.id) %>
+
```

Comment:
Is Hound asking for this extra line? Looks a little funny to my untrained :eyes: